packages/fuel-gauge/src/vectors.test.ts

```diff
@@ -34,6 +34,19 @@ describe('Vector Tests', () => {
     expect(value).toStrictEqual(INPUT);
   });
 
+  it('should test u8 vector input/output as Uint8Array', async () => {
+    using contractInstance = await setupContract();
+    const INPUT = [8, 6, 7, 5, 3, 0, 9];
+    const INPUT_AS_UINT8_ARRAY = Uint8Array.from(INPUT);
```

Comment:
```suggestion
    const INPUT: Vec<BigNumberish> = [8, 6, 7, 5, 3, 0, 9];
    const INPUT_AS_UINT8_ARRAY: Vec<BigNumberish> = Uint8Array.from(INPUT);
```

Is it worth adding the types here, to ensure they work as intended?

Reply:
@petertonysmith94 there's a separate issue for those types #2785, but I'm gonna be making changes to this PR regardless: https://github.com/FuelLabs/fuels-ts/pull/2884#discussion_r1704028350